packages/frp/build.sh

```diff
@@ -7,6 +7,7 @@ TERMUX_PKG_SRCURL=https://github.com/fatedier/frp/archive/v${TERMUX_PKG_VERSION}
 TERMUX_PKG_SHA256=af3e8d9d4144cf520cee2609cd45fb575afe711c03cc7441dc89d0402628a869
 # Depend on its subpackages.
 TERMUX_PKG_DEPENDS="frpc, frps"
+TERMUX_PKG_REVISION=1
```

Comment:
`VERSION` and `REVISION` should be adjacent to each other. Please keep in mind that packages are not only maintained by named maintainers.

Reply:
> Please keep in mind that packages are not only maintained by named maintainers.
I already knew, thanks.